packages/design-system-docs/src/pages/components/Review/review.example.html

```diff
@@ -1,8 +1,6 @@
-<div
-  class="ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex"
->
+<div class="ds-c-review">
   <div class="ds-u-margin-right--2">
-    <h3 class="ds-text ds-u-margin-bottom--0 ds-u-font-weight--bold ds-u-display--inline-block">
+    <h3 class="ds-c-review__heading ds-text">
```

Comment:
What is this additional `ds-text` for?

Reply:
Thanks for picking this up. Have updated.